pulse_actions/handlers/treeherder_resultset.py

```diff
@@ -25,6 +25,7 @@ def on_resultset_action_event(data, message, dry_run, stage=False):
     repo_name = data["project"]
     action = data["action"]
     times = data["times"]
+    priority = data["priority"]
```

Comment:
There is no priority data coming when calling this from Treeherder.
In order words, this won't work.